client/src/views/admin/dashboard.tsx

```diff
@@ -1,7 +1,7 @@
 const Dashboard = () => {
   return (
     <>
-      <h1 style={{ color: "#000" }}>ADMIN</h1>
+      <h1 style={{ color: "#3102ff" }}>ADMIN</h1>

```

Comment:
butu gerai cia tureti konstantas spalvu arba geriausia overridinti MUI tema ir imti is ten spalvas

Reply:
this is all temp